app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = {p_data["name"]: Person(p_data["name"], p_data["age"]) for p_data in people}
```

Comment:
The dictionary comprehension creates `Person` objects for each entry in the `people` list. This is correct, but ensure that the `people` list contains dictionaries with 'name' and 'age' keys to avoid KeyError.